src/internal/proc.js

```diff
@@ -487,22 +486,30 @@ export default function proc(
   function runForkEffect({context, fn, args, detached}, effectId, cb) {
     const taskIterator = createTaskIterator({context, fn, args})
 
-    asap.suspend()
-    const task = proc(taskIterator, subscribe, dispatch, getState, options, effectId, fn.name, (detached ? null : noop))
+    try {
```

Comment:
I see there is a new `try` here, but without corresponding `catch`, shouldnt something be done with the `error`? Also - what is it guarding from? `fn` throwing? 

this simple `try` can swallow errors easily and make debugging a nightmare